src/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -271,4 +271,27 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
 
         $queryBuilder->addOrderBy(\sprintf('%s.%s', $alias, $field), $direction);
     }
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function getSchema(Parameter $parameter): array
+    {
+        return ['type' => 'string', 'enum' => ['asc', 'desc']];
+    }
+
+    public function getOpenApiParameters(Parameter $parameter): OpenApiParameter|array|null
+    {
+        if (str_contains($parameter->getKey(), ':property')) {
+            $parameters = [];
+            $key = str_replace('[:property]', '', $parameter->getKey());
+            foreach (array_keys($parameter->getExtraProperties()['_properties'] ?? []) as $property) {
+                $parameters[] = new OpenApiParameter(name: \sprintf('%s[%s]', $key, $property), in: 'query');
+            }
+
+            return $parameters;
+        }
```

Comment:
let's put this in a trait inside doctrine/common

PropertyPlaceholderOpenApiParameterTrait